src/playercards/customizable/SummonedServitorUpgradeSheet.ttslua

```diff
@@ -178,4 +193,79 @@ function createButtonsAndBoxes()
             value          = box_data.value
         })
     end
+
+    makeSlotSelectionButtons()
 end
+
+-- Creates the invisible buttons overlaying the slot words
+function makeSlotSelectionButtons()
+    local buttonPositions = { x = -1 * SLOT_ICON_POSITIONS.arcane.x, y = 0.2, z = SLOT_ICON_POSITIONS.arcane.z }
+    local buttonData = {
+          click_function = "click_arcane",
+          function_owner = self,
+          position       = buttonPositions,
+          height         = 130,
+          width          = 130,
+          scale          = { 1, 1, 1 },
+          color          = { 0, 0, 0, 0 },
+    }
+    self.createButton(buttonData)
+    buttonData.click_function = "click_ally"
+    buttonPositions.x = -1 * SLOT_ICON_POSITIONS.ally.x
+    buttonData.position = buttonPositions
+    self.createButton(buttonData)
+  end
+
+function click_arcane()
+    if selectedSlot ~= "arcane" then
```

Comment:
Reverse this to avoid the negative logic

Reply:
To be clear, you prefer it like this?

```
    if selectedSlot == "arcane" then
        selectedSlot = ""
    else
        selectedSlot = "arcane"
    end
    updateSlotDisplay()
```

Reply:
Yep, that looks better.  It's best to avoid negative logic unless it's absolutely necessary for clarity

https://csharpdeveloper.wordpress.com/2016/01/20/roberts-rules-of-coders-8-avoid-negative-conditionals-and-method-names/

Reply:
Perfect, addressed!